universal-application-tool-0.0.1/app/services/question/ActiveAndDraftQuestions.java

```diff
@@ -0,0 +1,64 @@
+package services.question;
+
+import akka.japi.Pair;
+import com.google.common.collect.ImmutableMap;
+import com.google.common.collect.ImmutableSet;
+import com.google.common.collect.Sets;
+import java.util.Optional;
+import models.Question;
+import models.Version;
+import services.question.types.QuestionDefinition;
+
+public class ActiveAndDraftQuestions {
```

Comment:
Suggestion (blocking): please document this class

Reply:
Done - copied from Programs.